CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ rsyslog Cookbook CHANGELOG
 ==========================
 This file is used to list changes made in each version of the rsyslog cookbook.
 
+v.3.0.1 (2015-11-15)
```

Comment:
Can you remove the version bump.  This will most likely go into 4.0.0, but I have more work before that release can happen